src/main/xar-resources/data/urlrewrite/urlrewrite.xml

```diff
@@ -372,7 +422,7 @@
       throughout the entire pipeline, we need to clear the
       <literal>xslt.input</literal>
       with an explicit call to clear-attribute.</para>
-    <para>What benefits does it have to exchange data through request attributes: We save one serialization step:
+    <para>The benefit does to exchanging data through request attributes is that we save one serialization step.
```

Comment:
I suggest:

`The benefit of exchanging data through request attributes is that we save one serialization step.`